### DISCUSSION TITLE
How can a project be verified to the directory?

### DISCUSSION THREAD

**Discussion by chefallan** [web3phl/directory#52] [Q&A] [answered]
For example, they have submitted the form, and their project is added to the directory without the verified indication beside their project name. And they want to be verified on the directory to gain more trust in the community.

**Comment by warengonzaga** [comment, marked as answer]
Hey @itsnalla! That's a good question, right now the verification badge is for our partners only. We'll announce an update on how you can get your verification badge. More info soon!

I'll update this answer once the verification process starts.

**Reply by chefallan** [reply]
Thanks for the clarifications.
